src/robot/parsing/lexer/blocklexers.py

```diff
@@ -26,8 +26,7 @@
                               ForHeaderLexer, InlineIfHeaderLexer,
                               IfHeaderLexer, ElseIfHeaderLexer, ElseHeaderLexer,
                               TryHeaderLexer, ExceptHeaderLexer, FinallyHeaderLexer,
-                              EndLexer, ReturnLexer)
-
+                              EndLexer, ReturnLexer, ContinueLexer, BreakLexer)
 
 class BlockLexer(Lexer):
```

Comment:
An empty line has been accidentally removed. PEP-8 says there should be two such lines between imports and the class declaration.

Reply:
Corrected.